### PR TITLE
Initial commit to fog-google

### PR DESCRIPTION
**Pull request by danbroudy**
Copies files and takes a stab at setting up the gem.

### CODE REVIEW

**Comment by danbroudy** [comment]
@geemus I hope it was appropriate for me to include you in the spec.authors

I have never set up a gem before so any input is much appreciated.

@icco for visibility.

 I'm excited to get this going!

File: lib/fog/google/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * Project Metadata
+
+Features we are looking forward to implementing in the future:
+
+ * Image creation
+ * Load balancer configuration
+
+If you are using Fog to interact with GCE, please keep Fog up to date and [file issues](https://github.com/fog/fog/issues?labels=google) for any anomalies you see or features you would like.
+
+Thanks!
```
Comment (icco) [inline lib/fog/google/README.md]: I'd delete this and merge it into the main readme for this repo. But we can save that for another PR if you just want to have this be a clean import.

**Comment by icco** [comment]
If possible, it'd be nice if you could fix the author of the first commit. Give that a try, and if it doesn't work, we'll just merge and move on.

I haven't tried running this yet, but from a high level, it looks like a direct copy, so SGTM :+1:

**Comment by danbroudy** [comment]
Updated. The funny first commit author was me playing around with ways to signal I wasn't the original author of the files.

**Comment by geemus** [comment]
Thanks!

**Pull request merged**